app/components/tutorial.cjsx

```diff
@@ -42,15 +42,18 @@ module.exports = React.createClass
       TutorialComponent = this
 
       if tutorial.steps.length isnt 0
-        awaitTutorialMedia = tutorial.get 'attached_images'
-          .catch ->
-            # Checking for attached images throws if there are none.
-            []
-          .then (mediaResources) ->
-            mediaByID = {}
-            for mediaResource in mediaResources
-              mediaByID[mediaResource.id] = mediaResource
-            mediaByID
+        if tutorial.links.attached_images.ids?.length
```

Comment:
Ok, how about `if tutorial.links.attached_images.ids? and tutorial.links.attached_images.ids.length isnt 0`? so checks for existence and that the array has length.